templates/catalog/book/bookInfo.tpl

```diff
@@ -1,100 +1,120 @@
+ * --------------------------------------------------------------------
+ *}
+
+<script type="text/javascript">
+	// Attach the tab handler.
+	$(function() {ldelim}
+		$('#bookInfoTabs').pkpHandler('$.pkp.controllers.TabHandler');
+	{rdelim});
+</script>
+
+<div class="bookInfo">
+	<div class="bookInfoHeader">
+		<h3>{$publishedMonograph->getLocalizedFullTitle()|strip_unsafe_html}</h3>
+		<div class="authorName">{$publishedMonograph->getAuthorString()}</div>
+	</div>
+
+	<div id="bookInfoTabs" class="pkp_controllers_tab">
+		<ul>
+			<li><a href="#abstractTab">{translate key="submission.synopsis"}</a></li>
+			{if $chapters|@count != 0}<li><a href="#contentsTab">{translate key="common.contents"}</a></li>{/if}
+			{* Download-Tab deactivated
+			 {if $availableFiles|@count != 0}<li><a href="#downloadTab">{translate key="submission.download"}</a></li>{/if}
+			*}
+			{call_hook|assign:"sharingCode" name="Templates::Catalog::Book::BookInfo::Sharing"}
+			{if !is_null($sharingCode) || !empty($blocks)}
+				<li><a href="#sharingTab">{translate key="submission.sharing"}</a></li>
+			{/if}
+		</ul>
+
+		<div id="abstractTab">
+			{$publishedMonograph->getLocalizedAbstract()|strip_unsafe_html}
+
+			{assign var=authors value=$publishedMonograph->getAuthors()}
+			{foreach from=$authors item=author}
+				{if $author->getIncludeInBrowse()}
+					<p>{translate key="catalog.aboutTheAuthor" roleName=$author->getLocalizedUserGroupName()}: <strong>{$author->getFullName()}</strong></p>
+					{assign var=biography value=$author->getLocalizedBiography()|strip_unsafe_html}
+					{if $biography != ''}{$biography}{else}{translate key="catalog.noBioInfo"}{/if}
+				{/if}
+			{/foreach}
+		</div>
+		{if $chapters|@count != 0}
+			<div id="contentsTab">
+				{foreach from=$chapters item=chapter}
+					<p>
+						{* TODO: Move style to Stylesheet *} 
+						<div style='float:left;width:70%;margin-bottom:10px'>
+						<strong>{$chapter->getLocalizedTitle()}</strong>
+						{if $chapter->getLocalizedSubtitle() != '' }<br />{$chapter->getLocalizedSubtitle()}{/if}
+						{assign var=chapterAuthors value=$chapter->getAuthorNamesAsString()}
+						{if $publishedMonograph->getAuthorString() != $chapterAuthors}
+							<div class="authorName">{$chapterAuthors}</div>
+						{/if}
+						</div>
+						<div>
+							{assign var=publicationFormats value=$publishedMonograph->getPublicationFormats()}
+							{assign var=currency value=$currentPress->getSetting('currency')}							
+							{foreach from=$publicationFormats item=publicationFormat}
+								{if $publicationFormat->getIsAvailable()}
+									{include file="catalog/book/bookFilesContentAndDownload.tpl" availableFile=$availableFile publicationFormatId=$publicationFormat->getId() publishedMonograph=$publishedMonograph currency=$currency chapterId=$chapter->getId()}
+								{/if}
+							{/foreach}	
+						</div>
+					</p>
+				{/foreach}
+			</div>
+		{/if}
+{* old Download-Tab

```

Comment:
If this code is to be removed because of the changes you're proposing, please go ahead and remove it rather than just commenting it out.